addons/l10n_dk_oioubl/tests/test_xml_oioubl_dk.py

```diff
@@ -173,6 +176,18 @@ def test_export_partner_fr_without_siret_should_raise_an_error(self):
         with self.assertRaisesRegex(UserError, "The company registry is required for french partner:"):
             self.create_post_and_send_invoice(partner=self.partner_c)
 
+    @freeze_time('2017-01-01')
+    def test_oioubl_export_partner_without_vat_number(self):
+        """ This test verifies that we can't export an OIOUBL file for a partner
+            who doesn't have a tax ID. It verifies that we receive a UserError
+            telling to the user that this field is missing.
+        """
+        partner_without_vat_number = self.partner_b.copy()
```

Comment:
I don't really see why you create a new partner, why not using the existing one ? :)

Reply:
No real reason, thank you!